default/stringtables/en.txt

```diff
@@ -4705,6 +4705,29 @@ Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]] and the recen
 
 '''
 
+SITREP_DISMISS_HIDE_TITLE
+Dismissing and Hiding SitReps
+
+SITREP_DISMISS_HIDE_SHORT_DESC
+Dismissing a specific SitRep and hiding SitRep types.
+
+SITREP_DISMISS_HIDE_TEXT
+'''There are two options for ignoring Situation Reports (SitReps): Dismissing and Hiding.
+
+Dismissing a SitRep will keep that specific SitRep for displaying.
+There are options to dismiss for a certain number of turns, or indefinitely.
+When dismissing a SitRep, it will only stay dismissed for the current game.
+If you restart the client or reload the current game, these SitReps will display again.
+Dismissing is always for a specific SitRep, such as a reminder to build a Gas Generator on planet Delphi II.
```

Comment:
So if a fleet arrival sitrep for a particular fleet at a particular system, and the fleet arrives at that system again a few turns later, there'd be no sitrep? Maybe that starts to make sense, but I think you need to explicitly explain "that specific sitrep" immediately.